.landscaper/blueprint/deploy-executions.yaml

```
@@ -0,0 +1,36 @@
+deployItems:
+- name: deploy
+  type: landscaper.gardener.cloud/kubernetes-manifest
+  target:
+    name: {{ .imports.cluster.metadata.name }}
+    namespace: {{ .imports.cluster.metadata.namespace }}
+  config:
+    apiVersion: manifest.deployer.landscaper.gardener.cloud/v1alpha2
+    kind: ProviderConfiguration
+
+    updateStrategy: patch
+
+    manifests:
+      - apiVersion: core.gardener.cloud/v1beta1
```

the v1alpha2 has a different struct: 
```suggestion
    manifests:
      - policy: manage
         manifest: apiVersion: core.gardener.cloud/v1beta1
```